### ISSUE TITLE
Description for creating fed project

### ISSUE THREAD

**Issue by rayzhou2017** [kubesphere/console#688]
It is better to describe it is a fed project, for instance, here `Create Federated Project`
<img width="962" alt="Screen Shot 2020-07-09 at 1 15 50 PM" src="https://user-images.githubusercontent.com/28859385/86999983-82d72c80-c1e6-11ea-868c-d4d10781fdb0.png">

**Comment by Sherlock113** [comment]
I found this place in code:
<img width="993" alt="截屏2020-07-09下午2 47 15" src="https://user-images.githubusercontent.com/65327072/87006395-2fb7a680-c1f3-11ea-881f-ea7fb683f5f9.png">

I tried to change it and build it locally. It turned out that this UI text determines at least four places in console:

<img width="1092" alt="截屏2020-07-09下午2 56 40" src="https://user-images.githubusercontent.com/65327072/87007233-807bcf00-c1f4-11ea-9d61-192b1f772ce7.png">
<img width="1187" alt="截屏2020-07-09下午2 50 13" src="https://user-images.githubusercontent.com/65327072/87007270-8e315480-c1f4-11ea-908e-4221f0483a59.png">

In the interface to create projects, the UI "Create Project" will work for all these four places. Changing one means all of them will be changed. I think the logic to give different UI texts (Create Project vs Create Fed Project) is quite reasonable, but I think maybe we only need to change Create Project for Number 1 in the image? Others also include common projects that do not run on multiple clusters.
